content/en/users/tutorials/accessing-vm-with-ssh/_index.md

```diff
@@ -0,0 +1,220 @@
+---
+
+## Overview
+
+This page provides an introduction of connecting from a local computer to a
+cloud host via SSH. It provides general guidelines, SSH options, tips, and
+examples for setting up the OpenStack environment.
+
+## Username and password
+
+{{% alert title="Warning" color="warning" %}} `Username` and `password` access
+to cloud virtual machine images is usually **disabled** for security reasons and
+it is **strongly suggested not to be used**. {{% /alert %}}
+
+To enable SSH password authentication, the destination virtual machine needs to
+have changed `PasswordAuthentication no` to `PasswordAuthentication yes` in the
+`/etc/ssh/sshd_config` file.
+
+If really needed, a custom image with `PasswordAuthentication` enabled can be
+used or that can be injected when the virtual machine is deployed.
+
+Depending on your deployment method it could be done with Ansible, Terraform,
+Salt, Puppet, Chef, cloud-init, or your own deployment tool if supported (i.e.
+the Infrastructure Manager and a custom TOSCA template).
+
+{{% alert title="Warning" color="warning" %}} If you enable
+`PasswordAuthentication`, be sure to
+[generate a strong and unique password or passphrase](https://bitwarden.com/password-generator/)
+for your account, otherwise you virtual machines **will be** compromised, and
+you access may be suspended.{{% /alert %}}
+
+## SSH Keys
+
+A common method to access a cloud virtual machine is via ssh using **SSH keys**,
+you may inject your **public key** into the virtual machine, at deployment time,
+and use your **private key** to connect via ssh without any password.
+
+{{% alert title="Tip" color="info" %}} If you are using ssh keys in GitHub your
+public keys are available at:
+
+    https://github.com/${github_username}.keys
+
+i.e.:
+
+    wget https://github.com/github_username.keys
+
+{{% /alert %}}
+
+### SSH username
+
+The username to use to connect to a virtual machine is dependent on the virtual
+machine image and is generally different in each operative system image.
+
+For `official OS virtual machine images` you can use the general reference
+documentation on
+[obtaining images](https://docs.openstack.org/image-guide/obtain-images.html).
+
+For custom virtual machine images you need to refer to your virtual machine
+image provider (i.e. it could be something specific like `cloudadm`).
+
+It is also possible to change the username using cloud-init with a user-data
+configuration (i.e. see the
+[cloud config examples](https://cloudinit.readthedocs.io/en/latest/topics/examples.html))
+or inject some code to add additional users (i.e. with Ansible).
+
+### Local ssh key configuration
+
+The `private ssh-key` stored on your local computer is required to have
+restrictive file permissions. Depending on your local operative system you may
+need to run:
+
+```shell
+chmod 600 ~/.ssh/id_rsa
+```
+
+(with `id_rsa` being the name of the private key associated with the public key
+in use).
+
+## OpenStack networking
```

Comment:
This deserves a section of its own in the documentation and @thebe14 should be working on it.
@thebe14 any suggestions on how to deal with this?

Reply:
We could also merge this version and reword/change once we have another place where it's documented.
It's true that to get ssh working the VM needs to be configured appropriately, so it makes sense to have some stuff here.

Reply:
Let's assume we will change this once we have another page for this.